### ISSUE TITLE
cygwin path translation fail during box add

### ISSUE THREAD

**Issue by rct** [mitchellh/vagrant#6990]
Under Cygiwn Terminal (bash + MinTTY) adding a box from a *local* file fails.  It looks like after first detecting a local box file,  a later step decides it is remote and tries to turn it into an incomplete `file:///` URL.

```
$ vagrant box add cdkv2 /cygdrive/c/Downloads/rhel-cdk-kubernetes-7.2-13.x86_64.vagrant-virtualbox.box
==> box: Box file was not detected as metadata. Adding it directly...
==> box: Adding box 'cdkv2' (v0) for provider:
    box: Downloading: /cygdrive/c/Downloads/rhel-cdk-kubernetes-7.2-13.x86_64.vagrant-virtualbox.box
    box:
An error occurred while downloading the remote file. The error
message, if any, is reproduced below. Please fix this error and try
again.

Couldn't open file /C:/cygdrive/c/Downloads/rhel-cdk-kubernetes-7.2-13.x86_64.vagrant-virtualbox.box
```

The work around is to be in the same directory as the box file so no path is specified.

```
$ vagrant box add cdkv2 rhel-cdk-kubernetes-7.2-13.x86_64.vagrant-virtualbox.box
==> box: Box file was not detected as metadata. Adding it directly...
==> box: Adding box 'cdkv2' (v0) for provider:
    box: Unpacking necessary files from: file://C:/Downloads/rhel-cdk-kubernetes-7.2-13.x86_64.vagrant-virtualbox.box
    box:
==> box: Successfully added box 'cdkv2' (v0) for 'virtualbox'!

**Comment by rct** [comment]
See also #5288.  I reported the same or similar, which was marked fixed in 1.8.0

**Comment by chrisroberts** [comment]
Closing as fixed via #5288
